qiskit/transpiler/instruction_durations.py

```diff
@@ -170,19 +169,10 @@ def update(self, inst_durations: "InstructionDurationsType" | None, dt: float =
 
         return self
 
-    @deprecate_arg(
-        "qubits",
-        deprecation_description=(
-            "Using a Qubit or List[Qubit] for the ``qubits`` argument to InstructionDurations.get()"
-        ),
-        additional_msg="Instead, use an integer for the qubit index.",
-        since="0.19.0",
-        predicate=_is_deprecated_qubits_argument,
```

Comment:
I think `_is_deprecated_qubits_argument` (def in line 27) can also be removed, if this is the only usage.

Reply:
Thanks! Removed `_is_deprecated_qubits_argument` as you suggested.